docs/2_fitting.rst

```diff
@@ -2,8 +2,636 @@
+                                   each:
+                                      md: 10
+                                velocities:
+                                   each:
+                                      md: 10
+                                forces:
+                                   each:
+                                      md: 10
+                            md:
+                                ensemble: NVT
+                                temperature: 300.0
+                                timestep: 1
+                                steps: 10000
+                                thermostat:
+                                    type: csvr
+                                    csvr:
+                                        timecon: 10000
+                                print:
+                                    energy:
+                                        each:
+                                           md: 10
+        
+        monte_carlo:
+            type: FOX.armc.ARMC
+            iter_len: 50000
+            sub_iter_len: 10
+            logfile: armc.log
+            hdf5_file: armc.hdf5
+            path: ./
+            folder: MM_MD_workdir
+            keep_files: True
+
+The yaml code above shows a clear resemblance to the one used for the core, except for a few key differences. We hereby provide a brief comparison of their features.
+
+The param block
+---------------
+
+    .. code:: yaml
+    
+        param:
+            charge:
+                param: charge
+                Cs: 0.4
+                Pb: 0.8
+                Br: -0.4
+                C2O3: 0.25
+                O2D2: -0.275
+                constraints:
+                    - '0 < Cs < 1.5'
+                    - '0 < Pb < 2'
+                    - '-1.5 < Br < 0'
+                    - 'Cs == -1 * $LIGAND'
+                    - 'Pb == -2 * $LIGAND'
+                    - 'Cs == -1 * Br'
+                    - 'Pb == -2 * Br'
+                    
+          
+Here, the Coulomb potential sub-block shows both the charges of the nanocrystal core ions (Cs, Pb, Br) and those of the ligand anchoring group atoms (in this specific case, the carboxylate group of the acetate, i.e. C2O3 and O2D2). Their initial values are usually obtained: 
+    
+    * For the nanocrystal core ions, from the approximated results of the previous fitting procedure used for the inorganic core or by their most stable oxidation state, in absence of more accurate parameters.
+    * For the anchoring group of the ligand, by adjusting the charges (found both in the .yaml input and in the .rtf file of the ligand) to achieve the overall charge neutrality of the system. More specifically, the total charge of the ligand needs to equal the charge of the atom it replaces: in this specific case, our ligand is an acetate group, and it thus needs to balance the charge of the Br atom (-0.4). We will provide an example of this procedure in the following section.
```

Comment:
```suggestion
    * For the anchoring group of the ligand, by adjusting the charges (found both in the .yaml input and in the CHARMM .rtf file of the ligand) to achieve the overall charge neutrality of the system. More specifically, the total charge of the ligand needs to equal the charge of the atom it replaces: in this specific case, our ligand is an acetate group, and it thus needs to balance the charge of the Br atom (-0.4). We will provide an example of this procedure in the following section.
```
Explicitly mentioning CHARMM when talking about the .rtf format might be useful, as to not confuse it with the more general rich text format.